global-shared-library/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -0,0 +1,66 @@
+<archetype-descriptor xmlns="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0"
+                      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                      xsi:schemaLocation="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0 http://maven.apache.org/xsd/archetype-descriptor-1.0.0.xsd"
+                      name="Global Shared Library Example">
+    <requiredProperties>
+        <requiredProperty key="groupId">
+            <defaultValue>unused</defaultValue>
+        </requiredProperty>
+        <requiredProperty key="package">
+            <defaultValue>org.jenkinsci.pipeline.sample</defaultValue>
+        </requiredProperty>
+    </requiredProperties>
+    <fileSets>
+        <fileSet filtered="true">
+            <directory>.</directory>
+            <includes>
+                <include>pom.xml</include>
+            </includes>
+        </fileSet>
+        <fileSet filtered="true">
+            <directory></directory>
```

Comment:
`.`?

Reply:
It is odd but : https://stackoverflow.com/a/15435608